src/anypytools/abcutils.py

```diff
@@ -443,6 +444,43 @@ def save_results(self, filename, append=False):
         else:
             raise ValueError('Noting to save')
 
+    def save_to_hdf5(self, filename: str, batch_name: str):
+        if self.cached_tasklist:
+            any_output = [task.get_output() for task in self.cached_tasklist]
+            any_output = AnyPyProcessOutputList(any_output)
+            with h5py.File(filename, "w") as f:  # , compression="gzip" # Not sure how much gzip effects reading speed.
+                group = f.create_group(batch_name)
+                task_names = []
```

Comment:
Det kunne være skrevet kort: `task_names = [ r['task_name'] for r in any_output]`